openhands/server/routes/secrets.py

```diff
@@ -75,7 +75,8 @@ async def check_provider_tokens(
     if incoming_provider_tokens.provider_tokens:
         # Determine whether tokens are valid
         for token_type, token_value in incoming_provider_tokens.provider_tokens.items():
-            if token_value.token:
```

Comment:
I think this file can be reverted entirely; adding new providers should still be compatible with these endpoints as the existing logic meant to handle two or more providers